src/components/ClientForm.vue

```diff
@@ -96,20 +137,59 @@
       </div>
       <div class="field field--half">
         <label>
-          <span>Company Facebook</span>
-          <input v-model="companyFacebook" type="text" />
+          <span>
+            Company LinkedIn
+            <a
+              v-if="client.get('company').get('linkedin')"
+              :href="
+                `https://www.linkedin.com/company/${client
+                  .get('company')
+                  .get('linkedin')}`
+              "
+              target="_blank"
+            >
+              <font-awesome-icon :icon="['fab', 'linkedin']" />
+            </a>
+          </span>
+          <input v-model="companyLinkedin" type="text" />
         </label>
       </div>
       <div class="field field--half">
         <label>
-          <span>Company Instagram</span>
-          <input v-model="companyInstagram" type="text" />
+          <span>
+            Company Facebook
+            <a
+              v-if="client.get('company').get('facebook')"
+              :href="
+                `https://www.facebook.com/${client
+                  .get('company')
+                  .get('facebook')}`
+              "
+              target="_blank"
+            >
+              <font-awesome-icon :icon="['fab', 'facebook-square']" />
+            </a>
+          </span>
+          <input v-model="companyFacebook" type="text" />
         </label>
       </div>
       <div class="field field--half">
         <label>
-          <span>Company LinkedIn</span>
-          <input v-model="companyLinkedin" type="text" />
+          <span>
+            Company Instagram
+            <a
+              v-if="client.get('company').get('instagram')"
+              :href="
+                `https://www.instagram.com/${client
+                  .get('company')
+                  .get('instagram')}`
+              "
+              target="_blank"
+            >
+              <font-awesome-icon :icon="['fab', 'instagram-square']" />
```

Comment:
I prefer the original instagram icon rather than the square one, it just looks weird.